tools/clang/test/CodeGenSPIRV/vk.spec-constant.attributes.derivative-group.hlsl

```diff
@@ -0,0 +1,35 @@
+// RUN: %dxc -T cs_6_6 -E main -fspv-extension=SPV_NV_compute_shader_derivatives -fcgl %s -spirv 2>&1 | FileCheck -check-prefix=CHECK-LINEAR %s
+// RUN: %dxc -T cs_6_6 -E main -fspv-extension=SPV_NV_compute_shader_derivatives -fcgl -DQUADS %s -spirv 2>&1 | FileCheck -check-prefix=CHECK-QUADS %s
+
+// Note: validation disabled until NodePayloadAMDX pointers are allowed
+// as function arguments
+
+// CHECK-LINEAR: OpCapability ComputeDerivativeGroupLinearKHR
+// CHECK-LINEAR: OpExecutionMode %{{[^ ]*}} DerivativeGroupLinearKHR
+// CHECK-QUADS: OpCapability ComputeDerivativeGroupQuadsKHR
+// CHECK-QUADS: OpExecutionMode %{{[^ ]*}} DerivativeGroupQuadsKHR
```

Comment:
Looks like we are making a guess based on the default values of the spec constants. That seems reasonable.